platform/security/core/security-core-impl/src/main/java/ddf/security/config/impl/ConfigurationSecurityLogger.java

```diff
@@ -75,8 +83,17 @@ public void configurationEvent(ConfigurationEvent event) {
   }
 
   @SuppressWarnings("squid:S00112" /* Generic RuntimeException is acceptable here */)
-  private String dictionaryToString(Dictionary<String, Object> properties) {
+  @VisibleForTesting
+  String dictionaryToString(Dictionary<String, Object> properties) {
     ByteArrayOutputStream baos = new ByteArrayOutputStream();
+
+    if (properties.remove(PASSWORD) != null) {
```

Comment:
❓ How do we know these are the only 2 password properties that should be sanitized? I though this info was stored in the meta types.